tests/apis/prv_api/test_vehicles.py

```diff
@@ -104,7 +105,6 @@ def test_device_list_basic(client, django_assert_num_queries):
     n = BASE_NUM_QUERIES
     n += 1  # query on devices
     n += 1  # query to get areas of device
-    n += 1  # query latest_event
```

Comment:
We don't even make the extra query in this new version? Does it mean it's even faster?